Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-current-contest.tsx

```diff
@@ -240,12 +236,15 @@ const CurrentContestsProvider = ({ children }: ICurrentContestsProviderProps) =>
             return;
         }
 
+        if (!isNil(startContestError)) {
+            setContestError(startContestData as unknown as IErrorDataType);
```

Comment:
Can this be removed 'as unknown as '

Reply:
It cannot be cast directly to IErrorDataType. : /